CRM/Contact/BAO/Group.php

```diff
@@ -515,7 +515,7 @@ public static function createSmartGroup(&$params) {
       if (isset($ssParams['saved_search_id'])) {
         $ssParams['id'] = $ssParams['saved_search_id'];
       }
-
+      $params['form_values'] = $params['formValues'];
```

Comment:
This function is only ever called from the unit tests! We could probably move it over to the test class